.idea/workspace.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please git rm --cache .idea 
this directory before next commit

You can always do a git status before git push, 
and use git rm --cache to the files or dirs you want to exclude